CONTRIBUTING.md

```diff
@@ -58,6 +58,7 @@ We welcome icon requests. Before you submit a new issue please make sure the ico
     - Governmental agencies, programs, departments
        - Allowed: International organizations and NGOs with supranational interests
        - Allowed: Space agencies
+    - Military-related brands and organizations
```

Comment:
```suggestion
    - Military-related brands and organizations
    - Governmental agencies, programs, departments
       - Allowed: International organizations and NGOs with supranational interests
       - Allowed: Space agencies
```

I'd move military above the government bit - just to be explicit about it earlier.